nntrainer/src/bn_layer.cpp

```diff
@@ -52,20 +52,19 @@ int BatchNormalizationLayer::initialize() {
       axes_to_reduce.push_back(i);
   }
 
-  Tensor mu = Tensor(dim);
-  Tensor var = Tensor(dim);
-  Tensor gamma = Tensor(dim);
-  Tensor beta = Tensor(dim);
-
-  mu.setZero();
-  var.setValue(1);
-  beta.setZero();
-  gamma.setValue(1);
+  Tensor mu =
+    initializeWeight(dim, initializers[static_cast<int>(BNParams::mu)], status);
```

Comment:
check the value of status?

Reply:
I am actually curious about the need for passing status here. Internally, `initializeWeight` is not modifying status.  Semantics could change though, do you think we need to pass the status for this function?

Reply:
lets change the functiona signature in another PR